src/main/webapp/i18n/en/statistics.json

```diff
@@ -10,20 +10,21 @@
         "title": "User statistics",
         "days": " days",
         "loggedInUsers": "Amount of logins of distinct users in the last ",
-        "activeUsers": "Amount of logins of active users in the last ",
         "totalSubmissions": "Total amount of submissions in the last ",
-        "exercises": {
-            "released": "Total amount of released exercises in the last ",
-            "due": "Total amount of exercise deadlines in the last "
-        },
         "conductedExams": "Total amount of conducted exams in the last ",
         "examParticipation": " -> amount of student who participated: ",
         "examRegistrations": " -> with the amount of students being registered in these exam: ",
         "activeTutors": "Total amount of tutors which have created an assessment in the last ",
         "results": "Total amount of results created in the last ",
         "resultFeedbacks": "-> feedback count: ",
         "userLogins": "User Logins",
-        "submissions": "Submissions"
+        "SUBMISSIONS": "Submissions",
+        "ACTIVE_USERS": "Active users",
+        "RELEASED_EXERCISES": "Released exercises",
```

Comment:
Why are those ALL CAPS? Maybe group those strings.

Reply:
This was due to the fact that i reference these values via enums. I adjusted it, they are lowercase now ([here](https://github.com/ls1intum/Artemis/pull/2511/commits/29286d4a70141478ca0c1481b0295e4ef7abc44b))